changelogs/fragments/71461_tty_wrapping.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
```

Comment:
This looks like a feature, not a bug.

Reply:
The ever conundrum, someone's bug is some else feature... depends who you ask. Marketing teams are more like to see features than admitting existence of bugs ;)

If that is the only issue with it I don't mind making it a feature. I am ok with any measure that would disable pre-wrapping of logging at more or less random intervals, which makes impossible reliably process the output.

Reply:
If this wasn't advertised as working, it cannot be a bug.